src/content-helper/editor-sidebar/editor-sidebar.scss

```diff
@@ -186,6 +186,365 @@
 	}
 }
 
+.wp-parsely-content-helper .wp-parsely-performance-panel {
+	padding: calc(16px);
+	border-top: 1px solid rgb(221, 221, 221);
+	margin-top: -1px;
+
+	.panel-body {
+		width: 100%;
+
+		/** View in Parse.ly button */
+		.components-button.wp-parsely-view-post {
+			display: flex;
+			justify-content: center;
+			align-items: center;
+			width: 100%;
+		}
+	}
+
+	.performance-stat-panel {
+		display: flex;
+		padding: to_rem(6px) 0 to_rem(16px) 0;
```

Comment:
I don't have a strong opinion on this, other than the hassle of having to use `to_rem` everytime I add a dimension. But using rem units is a good practice, I believe. 

As a note, it is possible that I have missed some `is_rem` in some dimensions, since some of the CSS was taken from Figma, and I didn't add the `is_rem` until I had it tested and working as we wanted. I will take a closer look and see if I didn't missed any :)

Reply:
I did some improvements on the CSS, adding `to_rem` to everywhere I missed it, and adding a few extra variables.

Reply:
I think that the rem vars you've introduced are useful. This can be a way towards using real rem values.